src/utils/opcuaclient.js

```diff
@@ -109,13 +109,30 @@ const monitorVariable = async (subscription, item) => {
 
 const send = async (name, value) => {
   if (EGRESS_URLS) {
-    await fetch(EGRESS_URLS, {
-      method: 'POST',
-      body: JSON.stringify({
-        timestamp: Date.now(),
-        value,
-        name,
-      }),
+    const urls = []
+    const eUrls = EGRESS_URLS.replace(/ /g, '')
+    if (eUrls.indexOf(',') !== -1) {
```

Comment:
is this check really needed? I think we can just call `split`

Reply:
replaced it.

Reply:
you can probably save one line by doing something like
```
const urls = eUrls.split(',')
```